tests/fast-integration/test/5-deprovision-skr.js

```diff
@@ -0,0 +1,29 @@
+const {
+    KEBClient,
+    deprovisionSKR,
+    ensureOperationSucceeded,
+} = require("../keb");
+const { debug, genRandom } = require("../utils");
+
+
+describe("Provisioning SKR", function () {
+
+    const kebHost = process.env["KEB_HOST"] || "";
```

Comment:
I think you should encapsulate all those consts to some object. It would be easier to pass it around.